stl/inc/cmath

```diff
@@ -1330,6 +1374,36 @@ _NODISCARD constexpr _Ty _Common_lerp(const _Ty _ArgA, const _Ty _ArgB, const _T
             }
         }
 
+        auto _Is_Overflow = [](_Ty val) constexpr {
+#ifdef _M_FP_STRICT
+            if (_STD is_constant_evaluated()) {
+                return _Is_inf(val);
+            }
+            return fetestexcept(FE_OVERFLOW) != 0;
+#else // ^^^ defined(_M_FP_STRICT) / !defined(_M_FP_STRICT) vvv
+            return _Is_inf(val);
+#endif // _M_FP_STRICT
+        };
+
+        auto _Clear_FE_OVERFLOW_flag = []() constexpr {
+#ifdef _M_FP_STRICT
+            if (!_STD is_constant_evaluated()) {
+                feclearexcept(FE_OVERFLOW);
+            }
+#endif // _M_FP_STRICT
+        };
+
+        auto _Signbit = [](_Ty val) constexpr {
+            // std::signbit is not constexpr so cannot use :(
+            return val < 0;

```

Comment:
Is this a legitimate way to check the sign bit? I haven't looked at the rest of the code to determine if it matters.

Reply:
[Maybe  just copy our testing utils?](https://github.com/microsoft/STL/blob/62137922ab168f8e23ec1a95c946821e24bde230/tests/std/tests/GH_000935_complex_numerical_accuracy/floating_point_utils.hpp#L44)

Reply:
It seems only for -0.0 the functions return different results.
```
signbit(-0.0) = 1
-0.0 < 0 = 0
```
And it doesn't matter.
But sure, we can copy the existing code.

Reply:
Sorry for the bother 😀. Like I said, I didn't really check if that would cause a problem.